model/pdata/common.go

```diff
@@ -202,12 +202,12 @@ func (a AttributeValue) MapVal() AttributeMap {
 // such empty array has no effect on this AttributeValue.
 //
 // Calling this function on zero-initialized AttributeValue will cause a panic.
-func (a AttributeValue) ArrayVal() AnyValueArray {
+func (a AttributeValue) ArrayVal() AttributeSlice {
```

Comment:
SliceVal? I want consistency, maybe AttributeArray for the struct? Not sure what is best but I want consistency 👯

Reply:
Changed to `SliceVal()` and also fixed a couple comments still referring to `AnyValueArray`.